ta/pkcs11/src/pkcs11_helpers.h

```diff
@@ -55,6 +55,9 @@ bool key_type_is_asymm_key(uint32_t key_type_id);
 /* Boolprop flag shift position if @attribute_id is boolean, else -1 */
 int pkcs11_attr2boolprop_shift(uint32_t attribute_id);
 
+/* Return true if @attribute_id is boolean, else false */
+bool pkcs11_is_attr_bool(uint32_t attribute_id);
```

Comment:
What about this?
```c
/* Return true if @attribute_id is boolean, else false */
static inline bool pkcs11_is_attr_bool(uint32_t attr)
{
	return pkcs11_attr2boolprop_shift(attr) >= 0;
}
```

Reply:
Thanks. Removed the duplicate function pkcs11_is_attr_bool()